scripts/generate_builtins.py

```diff
@@ -135,8 +137,7 @@ def trimext(file: str) -> str:
                 else:
                     example_code = [override]
             elif not example_code:
-                raise Exception(f"No examples found for {function_name}")
-
+                continue
```

Comment:
If we couldn't parse the code of the function, just skip. I don't think parsing the docs like this is reliable and is probably easier to just handwrite the functions we want to support 🤔

Reply:
> I don't think parsing the docs like this is reliable and is probably easier to just handwrite the functions we want to support 

Yes that's what the overrides list is for. But we need this error to tell us when something needs to be added to the overrides list. This should error.

Reply:
Alright, reverted my changes to this generator and added a test program for `jsonencode` ✅